daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/value/Value.scala

```diff
@@ -344,30 +339,19 @@ object Value extends CidContainer1WithDefaultCidResolver[Value] {
 
   type NodeIdx = Int
 
-  /** Possibly relative contract identifiers.
-    *
-    * The contract identifiers can be either absolute, referring to a
-    * specific instance in the contract store, or relative, referring
-    * to a contract created in the same transaction and hence not yet
-    * having been assigned an absolute identifier.
-    *
-    * Note that relative contract ids are useful only before commit, in
-    * the context of a transaction. After committing we should never
-    * mention them.
-    *
-    * Why put it here and not just in Transaction.scala? Because we want
-    * to be able to use AbsoluteContractId elsewhere, so that we can
-    * automatically upcast to ContractId by subtyping.
-    */
-  sealed abstract class ContractId extends Product with Serializable
+  @deprecated("Use ContractId", since = "1.1.2")
```

Comment:
What's our process for removing deprecated elements such as this? Do we remove them in e.g. the next major SDK version?

Reply:
We do not have clearly defined process. My intention was to let this deprecated warning for a least one release.

Reply:
Sounds good.